<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDlgDistributeGame</class>
 <widget class="QDialog" name="CDlgDistributeGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>419</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>558</width>
    <height>419</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>558</width>
    <height>419</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Package Distribution Options</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>360</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="eRuntime">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>84</y>
     <width>421</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRuntime">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>84</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rZip">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>164</y>
     <width>251</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Zip archive (most compatible)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="r7z">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>194</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>7z archive (best compression)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rSFX">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>224</y>
     <width>291</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Self-extracting executable  (Windows only)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>54</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>LGCK runtime executable</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSFX">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>274</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="eSFX">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>274</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>93</x>
     <y>250</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>7zip SFX module</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>471</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Output Package</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>451</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Please select the distribution options that best suit your game's needs.</string>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>buttonBox</zorder>
  <zorder>eRuntime</zorder>
  <zorder>btnRuntime</zorder>
  <zorder>rZip</zorder>
  <zorder>r7z</zorder>
  <zorder>rSFX</zorder>
  <zorder>label</zorder>
  <zorder>btnSFX</zorder>
  <zorder>eSFX</zorder>
  <zorder>label_3</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CDlgDistributeGame</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CDlgDistributeGame</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
